docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
+
+- It is present in the same namespace as the TKG Cluster
+
+- It is associated with the same `StorageClass` as the default storage class in the TKG Cluster
+
+- Not attached to a Pod in the Project Pacific Cluster
+
+Steps to create a static PVC in a TKG Cluster are as follows:
+
+1. Note the name of the PVC in the Project Pacific cluster
+
+2. Create a PersistentVolume with the `storageClassName` set to the default storage class of the TKG Cluster and the `volumeHandle` pointing to the value retrieved in step 1.
```

Comment:
no default storage class in tkg

Reply:
@shalini-b shouldn't the storageClassName be set to the storage class name used by supervisor PVC?

Reply:
@SandeepPissay I have mentioned that is it preferred to use the same storage class as the Supervisor PVC.
@sashrith I have removed the default keyword.